src/data-store.js

```diff
@@ -47,6 +47,12 @@ function readSettings(settingKey) {
 // Setup/check the .linear config files
 function init() {
 	return new Promise(resolve => {
+		// does the user have a .linear file (not a directory)?
+		if (shell.test('-f', settingsPath())) {
+			// remove the .linear file from older installations of linear
+			shell.rm('-rf', settingsPath());
+		}
+
```

Comment:
this will remove the old `.linear` file before installing the `.linear` directory with the default config file and themes.